### PR TITLE
Add favicon, increase size for range selector.

### PR DESCRIPTION
**Pull request by bharath6365**
Description
Add favicon for the application. (Uses the same as that of the parent domain). After this change favicons won't work locally but I guess that's no big deal.
Increase the size of input type number thereby giving users more room to click. I do it only for the first fold and not for Severity Assumptions as there's no room if the input exceeds a certain value. Hence the CSS scoping.

number-size-increase

Impacted Areas in the application
Favicon
Forms

### CODE REVIEW

File: src/styles/global.scss [hunk 1]
```diff
@@ -86,6 +86,17 @@ p {
 input[type='number']::-webkit-inner-spin-button,
 input[type='number']::-webkit-outer-spin-button {
   opacity: 1;
+  
+}
+
+.form-group {
+  input[type='number']::-webkit-inner-spin-button,
```
Comment (ivan-aksamentov) [inline src/styles/global.scss]: This makes spin buttons disappear on Chrome. Could you please remove this?

**Comment by ivan-aksamentov** [comment]
@bharath6365 favicons are welcome.
The input style changes cause arrow buttons to disappear. This is not good.

**Comment by bharath6365** [comment]
@ivan-aksamentov  Did you get a chance to check?

**Comment by ivan-aksamentov** [comment]
@bharath6365 yep, see comments above.

**Comment by ivan-aksamentov** [comment]
@bharath6365 It's been a while without updates. I will close this for now for logistics purposes, but feel free to open another PR with the changes above. Thanks!

**Pull request closed without merging**